PySignal.py

```diff
@@ -2,16 +2,70 @@
 __copyright__ = "Copyright 2016, Dhruv Govil"
 __credits__ = ["Dhruv Govil", "John Hood", "Jason Viloria", "Adric Worley", "Alex Widener"]
 __license__ = "MIT"
-__version__ = "1.1.1"
+__version__ = "1.1.3"
```

Comment:
Bumped 2 minor changes (1 for sender, 1 for WeakMethod backport). 
Not committed, can change version number as you see fit.